src/ansible_runner/__main__.py

```diff
@@ -866,6 +872,10 @@ def main(sys_args=None):
 
         with context:
             with role_manager(vargs) as vargs:
+                if vargs.get('suppress_env_print'):
+                    suppress_env_print = vargs.get('suppress_env_print')
+                else:
+                    suppress_env_print = os.getenv('SUPPRESS_ENV_PRINT', 'False') == 'True'
```

Comment:
```suggestion
                    suppress_env_print = os.getenv('SUPPRESS_ENV_PRINT', 'False').lower() == 'true'
```

This will match all `True`, `true` and other combinations of cases.
Not a blocker though.